templates/node_inspector.html

```diff
@@ -1,12 +1,27 @@
 <div id="nodeInspector" class="container px-0">
     <!-- ko if: selectedNode() != null -->
 
-        {% include 'node_inspector_attributes.html' %}
-
+        <h5 class="card-header" id="nodeInspectorHeading" data-bind="click: $root.spinCollapseIcon, click: $root.toggleAllNodeMenus">
```

Comment:
I think this section of HTML could be split out into a new file "node_inspector_header.html" or similar. Just so that we follow the pattern of each section of the node_inspector having its own file.

Lots of small files is usually preferable to one large file for this sort of thing.